tests/drivers/build_all/sensor/sensors_trigger_global.conf

```diff
@@ -36,6 +36,7 @@ CONFIG_LIS2DH_TRIGGER_GLOBAL_THREAD=y
 CONFIG_LIS2DE12_TRIGGER_GLOBAL_THREAD=y
 CONFIG_LIS2DS12_TRIGGER_GLOBAL_THREAD=y
 CONFIG_LIS2DU12_TRIGGER_GLOBAL_THREAD=y
+CONFIG_LIS2DUX12_TRIGGER_GLOBAL_THREAD=y
```

Comment:
This is good. But I think you can also add lis2dux12 here: sensors_die_temp.conf

Reply:
Alright. I added it there as well.